lib/article-tile/index.js

```diff
@@ -2,7 +2,7 @@ import React, { Component, PropTypes } from 'react';
 
 require('./style.css');
 
-export class ArticleTile extends Component {
+export default class ArticleTile extends Component {
```

Comment:
I'm not 100% on ES6 modules, but I was looking at https://github.com/numo-labs/isearch-ui/pull/209/files#diff-fea5791e56ba08ff0d11c5afeac3eec3L5 and trying to work out why one of the things was not like the others, and the article tile was the only tile component that did not export a default.